src/main/java/net/raphimc/viabedrock/protocol/rewriter/ItemRewriter.java

```diff
@@ -120,9 +121,14 @@ public Item javaItem(final BedrockItem bedrockItem) {
                     rewriter = metaItemRewriter.get(meta);
                 }
             } else {
-                ViaBedrock.getPlatform().getLogger().log(Level.WARNING, "Missing bedrock -> java item mapping for " + identifier);
+                // ViaBedrock.getPlatform().getLogger().log(Level.WARNING, "Missing bedrock -> java item mapping for " + identifier);
                 final StructuredDataContainer data = ProtocolConstants.createStructuredDataContainer();
-                data.set(StructuredDataKey.ITEM_NAME, TextUtil.stringToNbt("§cMissing item: " + identifier));
```

Comment:
Add this as well into the else block. The item is still missing if the user didn't load the pack